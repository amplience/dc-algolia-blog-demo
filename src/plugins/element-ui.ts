import Vue from 'vue';
import { Card, Row, Col, Container, Header, Main, Pagination } from 'element-ui';

export default (): void => {
  Vue.use(Card);
  Vue.use(Row);
  Vue.use(Col);
  Vue.use(Container);
  Vue.use(Header);
  Vue.use(Main);
  Vue.use(Pagination);
};
