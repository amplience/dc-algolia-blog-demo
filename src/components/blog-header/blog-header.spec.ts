import { shallowMount } from '@vue/test-utils';
import { Header as ElHeader } from 'element-ui';
import BlogHeader from './blog-header.vue';

describe('Header', (): void => {
  test('is a Card instance with card data', (): void => {
    const wrapper = shallowMount(BlogHeader, {
      stubs: { ElHeader },
      propsData: {}
    });

    expect(wrapper.isVueInstance()).toBeTruthy();
    expect(wrapper.element).toMatchSnapshot();
  });
});
