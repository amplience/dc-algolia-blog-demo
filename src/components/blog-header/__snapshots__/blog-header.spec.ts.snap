// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header is a Card instance with card data 1`] = `
<div>
  <div
    class="header-heading"
  >
    <a
      href="/"
    >
      
      Amplience Dynamic Content Algolia Blog
    
    </a>
  </div>
   
  <div
    class="header-subheading"
  >
    
    Demonstrating how to integrate Amplience Dynamic Content with Algolia to create a simple blog
  
  </div>
</div>
`;
